src/calculateRentalCost.js

```diff
@@ -27,8 +27,21 @@
  * @return {number}
  */
 
+const SHORT_TERM_DISCOUNT = 20;
+const LONG_TERM_DISCOUNT = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  const TOTAL_SUM = days * 40;
+
+  if (days >= 3 && days < 7) {
+    return TOTAL_SUM - SHORT_TERM_DISCOUNT;
+  }
+
+  if (days >= 7) {
```

Comment:
please extract those numbers in separate variables

Reply:
thank you, all done